.eslintignore

```diff
@@ -0,0 +1 @@
+source/header.js
```

Comment:
I added `.eslintignore` to prevent ESLint from re-formatting `header.js`. `header.js` contains the comment that is inserted above the bundle, and it shouldn’t be formatted. It’d be better to move the comment from `header.js` straight to `gulpfile.js`, but it’s another task.